### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by JamesH65**
Add a New Bug template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us fix your issue
+
+---
+
+**Is this the right place for my bug report?**
+This repository contains the Linux kernel used on the Raspberry Pi. If you believe that the issue you are seeing is kernel related, this is the right place. If not, we have other repositories for the GPU firmware (https://github.com/raspberrypi/firmware) and Raspberry Pi userland applications (https://github.com/raspberrypi/userland). If you simply have a question, then the Raspberry Pi forums are the best place for that (https://www.raspberrypi.org/forums).
+
```
Comment (popcornmix) [inline .github/ISSUE_TEMPLATE/bug_report.md:9]: Maybe also add that problems with distribution packages should be reported here: https://github.com/RPi-Distro/repo

Comment (JamesH65) [reply .github/ISSUE_TEMPLATE/bug_report.md:9]: Done

**Comment by JamesH65** [comment]
This PR has been copy edited, so can now be merged if people are happy with it.

**Comment by JamesH65** [comment]
Actually, the CE's hasn't propagated to the PR for some reason....wait a mo...

**Pull request merged**